hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/repository/FileDataRepositoryTest.java

```diff
@@ -93,4 +95,17 @@ void getFileForFeeSchedules() throws InvalidProtocolBufferException {
         final var actualBytes = fileDataRepository.getFileAtTimestamp(FEE_SCHEDULE_ENTITY_ID.getId(), expiry);
         assertThat(CurrentAndNextFeeSchedule.parseFrom(actualBytes)).isEqualTo(feeSchedules);
     }
+
+    @Test
+    void getNullBytesForLargeFile() {
+        domainBuilder
+                .fileData()
+                .customize(f -> f.fileData(feeSchedules.toByteArray())
+                        .entityId(LARGE_FILE_ENTITY_ID)
+                        .consensusTimestamp(expiry))
+                .persist();
+
+        final var actualBytes = fileDataRepository.getFileAtTimestamp(LARGE_FILE_ENTITY_ID.getId(), expiry);
+        assertNull(actualBytes);
```

Comment:
Prefer not to hardcode fields unless necessary or increase their scope. Also always use assertj.
```suggestion
        var fileId = EntityId.of(0L, 0L, 151L);
        var file = domainBuilder.fileData().customize(f -> f.entityId(fileId).persist();

        assertThat(fileDataRepository.getFileAtTimestamp(fileId.getId(), file.getConsensusTimestamp()).isNull();
```